javaScriptAJAX-basicConcept/basicXMLRequest/basicXML.js

```diff
@@ -0,0 +1,8 @@
+var request = new XMLHttpRequest();
```

Comment:
It's better to use const instead of let to prevent accidental reassignment of variables and promote code stability.